src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -43,6 +49,11 @@ type Props = {
     file: BoxItem,
     getApproverWithQuery?: Function,
     getAvatarUrl: GetAvatarUrlCallback,
+    getComment?: (
```

Comment:
Should we handle this case in `ActivitySidebar.fetchFeedItems` rather than inside ActivityFeed?

Reply:
That makes a lot of sense. I've moved the logic for fetching reply data to `ActivitySidebar` but I've placed it in `fetchFeedItemsSuccessCallback` as I didn't want to change the original purpose of `fetchFeedItems. It seemed better to put it in `fetchFeedItemsSuccessCallback` as I'm taking actions based on the success of the `fetchFeedItems`.